engine/src/vs_logging.cpp

```diff
@@ -32,7 +34,7 @@
 #include <boost/log/expressions.hpp>
 #include <boost/log/sources/severity_logger.hpp>
 #include <boost/log/sources/record_ostream.hpp>
-//#include <boost/log/sinks/sync_frontend.hpp>
+#include <boost/log/sinks/async_frontend.hpp>
```

Comment:
Note to self: I need to go back to sync_frontend rather than async, at least for the time being.

Reply:
Done ✔️